### ISSUE TITLE
Version depedency

### ISSUE THREAD

**Issue by charleston10** [seegno/angular-oauth2#66]
"Include angular-oauth2 and dependencies. (angular, angular-cookies and query-string)" 
What's the version of angular / angular-cookies ?

**Comment by ruipenso** [comment]
```
"angular": "^1.4.0",
"angular-cookies": "^1.4.0",
"query-string": "^1.0.0"
```

**Comment by charleston10** [comment]
Thanks
Can you send a link of angular-cookies?
I'm testing the module, because i've a problem with cookie..

i use
https://ajax.googleapis.com/ajax/libs/angularjs/1.4.5/angular.min.js
http://ajax.googleapis.com/ajax/libs/angularjs/1.4.5/angular-cookies.js

$cookies.putObject && $cookies.getObject not worked

Is worked with
$cookies.myObject = "teste"
console.log($cookies.myObject)// show teste

**Comment by ruipenso** [comment]
I'm using `1.4.5` in some projects and I don't have any issue. The links are good.

Here's the example in `angular-cookies`:

```
angular.module('cookiesExample', ['ngCookies'])
  .controller('ExampleController', ['$cookies', function($cookies) {
     // Retrieving a cookie
     var favoriteCookie = $cookies.get('myFavorite');
     // Setting a cookie
     $cookies.put('myFavorite', 'oatmeal');
 }]);
```

Send me a snippet of your code... I can try to help.

BTW, I recommend you to use bower.

**Comment by charleston10** [comment]
Code
....
< head></ head>
< body>
		
		<div data-ng-app="app" data-ng-controller="Main as vm">				
			TEST
			<label data-ng-bind="vm.test"/>
		</div>	
			
		<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.5/angular.min.js"></script>
		<script src="http://ajax.googleapis.com/ajax/libs/angularjs/1.4.5/angular-cookies.js"></script>		
		
		<script>
			(function(angular,undefined){
				angular
					.module('app',['ngCookies'])			
					.controller('Main',['$cookies',function($cookies){
						var vm = this;
						vm.test = 'test';
						
						//undefined
						$cookies.put('myFavorite', 'dataFavorite');						
                		var favoriteCookie = $cookies.get('myFavorite');						
						console.log(favoriteCookie);
						
						//undefined
						$cookies.putObject('myFavorite', 'dataFavorite');			
                		var favoriteCookieObj = $cookies.getObject('myFavorite');						
						console.log(favoriteCookieObj);
						
						//worked
						$cookies.cook = 'cookies';
						var cook = $cookies.cook;
						console.log(cook);
						
						 //undefined
					     var favoriteCookie = $cookies.get('myFavorite');					    
					     $cookies.put('myFavorite', 'oatmeal');						 
						 console.log(favoriteCookie);
		        
					}]);
			})(angular);
		</script>
	</body>
</html>

**Comment by ruipenso** [comment]
Are you using a local file?

Browsers, normally have cookies disabled for local files.

In your code,
```
//worked
$cookies.cook = 'cookies';
var cook = $cookies.cook;
console.log(cook);
```

isn't storing any cookies, it just added the `cook` property to the `$cookies` object.

**Comment by charleston10** [comment]
It's *"isn't storing any cookies, it just added the cook property to the $cookies object."*

I put on a apache and worked..
Thanks very much

**Comment by charleston10** [comment]
Code
....
< head></ head>
< body>
		
		<div data-ng-app="app" data-ng-controller="Main as vm">				
			TEST
			<label data-ng-bind="vm.test"/>
		</div>	
			
		<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.5/angular.min.js"></script>
		<script src="http://ajax.googleapis.com/ajax/libs/angularjs/1.4.5/angular-cookies.js"></script>		
		
		<script>
			(function(angular,undefined){
				angular
					.module('app',['ngCookies'])			
					.controller('Main',['$cookies',function($cookies){
						var vm = this;
						vm.test = 'test';
						
						//undefined
						$cookies.put('myFavorite', 'dataFavorite');						
                		var favoriteCookie = $cookies.get('myFavorite');						
						console.log(favoriteCookie);
						
						//undefined
						$cookies.putObject('myFavorite', 'dataFavorite');			
                		var favoriteCookieObj = $cookies.getObject('myFavorite');						
						console.log(favoriteCookieObj);
						
						//worked
						$cookies.cook = 'cookies';
						var cook = $cookies.cook;
						console.log(cook);
						
						 //undefined
					     var favoriteCookie = $cookies.get('myFavorite');					    
					     $cookies.put('myFavorite', 'oatmeal');						 
						 console.log(favoriteCookie);
		        
					}]);
			})(angular);
		</script>
	</body>
</html>

**Comment by charleston10** [comment]
ruipenso (y)
